### ISSUE TITLE
UI::HtmlDialog DevTools position and size bug

### ISSUE THREAD

**Issue by linying1991** [SketchUp/api-issue-tracker#800]
# Bug Reports

1. SketchUp Version: All Version
2. OS Platform: Windows10, Windows 11

The location and size of each opening of the `UI::HtmlDialog` console `DevTools` are preset, and the last time is not recorded.
Every time you use it, you have to adjust the position manually, which is very troublesome. I hope you can repair the bug of the record location.

**Comment by DanRathbun** [comment]
This is not part of the APIs. Please report this in the correct repository.

**Comment by thomthom** [comment]
Agree, would be nice if the Developer Tools window persisted it position and size.

**Comment by DanRathbun** [comment]
What is this DevTools window ?

**Comment by linying1991** [comment]
@DanRathbun Front-end code debugging tools provided by SketchUp.

![无标题](https://user-images.githubusercontent.com/38310238/181135899-f6eb9e7c-2aa7-48e1-860c-61e4ae3b46c2.png)

**Comment by DanRathbun** [comment]
OH! okay ... you mean the **Chromium** DevTools.

**Yes ... I have noticed this bug as well.**

Also, (on MS Windows) when the primary display is at 125% display scaling and the DevTools window opens (or it moved to) the secondary display at 100% scaling, the layout of the DevTools gets messed up.

**Comment by kalpana-ghodeswar** [comment]
This issue is resolved in version SU2025.0
